src/fireedge/src/modules/components/Forms/VmTemplate/CreateForm/Steps/ExtraConfiguration/context/userInputsAutocompleteDialog.js

```diff
@@ -0,0 +1,210 @@
+      <DialogTitle>Add autocomplete suggestion</DialogTitle>
+      <DialogContent>
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'flex-end',
+            alignItems: 'end',
+            mb: 2,
+          }}
+        >
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={fetchSuggestions}
+            disabled={isFetching}
+            startIcon={
+              isFetching ? (
+                <CircularProgress size={20} color="inherit" />
+              ) : (
+                <Download />
+              )
+            }
+          >
+            Official suggestions
+          </Button>
+        </Box>
+
+        <TextField
+          autoFocus
+          margin="dense"
+          label="Text"
+          type="text"
+          fullWidth
+          value={text}
+          onChange={(e) => setText(e.target.value)}
+          error={error && !text}
+          helperText={error && !text ? 'Text is required' : ''}
+          placeholder="Display value in list"
+        />
+        <TextField
+          margin="dense"
+          label="Value"
+          type="text"
+          fullWidth
+          value={value}
+          onChange={(e) => setValue(e.target.value)}
+          error={error && (!value || !validateValue(value))}
+          helperText={
+            error && (!value || !validateValue(value))
+              ? 'Value must be a single word containing only letters, numbers, or underscores'
+              : ''
+          }
+          placeholder="Value on template"
+        />
+        <Box sx={{ mt: 2 }}>
+          {items.map((item, index) => (
+            <Card
+              key={`${item.text}-${index}`}
+              variant="outlined"
+              sx={{
+                mb: 1,
+                display: 'flex',
+                alignItems: 'center',
+                padding: '8px',
+                borderRadius: '10px',
+                boxShadow: '0 2px 4px rgba(0,0,0,0.1)',
+              }}
+            >
+              <CardContent sx={{ flexGrow: 1, padding: '8px !important' }}>
+                <Typography variant="body2" component="div">
+                  {item.text}
+                </Typography>
+                <Typography variant="caption" color="text.secondary">
+                  {item.value}
+                </Typography>
+              </CardContent>
+              <CardActions sx={{ padding: '0' }}>
+                <IconButton
+                  edge="end"
+                  aria-label="delete"
+                  onClick={() => handleDelete(item.text)}
+                  size="small"
+                >
+                  <DeleteIcon />
+                </IconButton>
+              </CardActions>
+            </Card>
+          ))}
+        </Box>
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={handleClose}>Cancel</Button>
+        <Button onClick={handleSubmit}>Add</Button>
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

JSX element content is not internationalized. Application should support multiple languages through internationalization. Instead, utilize packages like &#39;i18next&#39; to internationalize your items.